sdk/webpubsub/azure-messaging-webpubsubservice/tests/test_send_requests.py

```diff
@@ -1,38 +0,0 @@
-import io
-
-from azure.messaging.webpubsubservice import WebPubSubServiceClient
-from azure.messaging.webpubsubservice.rest import *
-
-from azure.core.credentials import AzureKeyCredential
-
-def test_build_text_request():
-    request = build_send_to_all_request('hub', content='hello world', content_type='text/plain')
-
-    assert request.headers['content-type'] == 'text/plain'
-    assert request.content == 'hello world'
-
-def test_build_json_request():
-    client = WebPubSubServiceClient('https://www.microsoft.com/api', AzureKeyCredential('abcd'))
-    request = build_send_to_all_request('hub', json={'hello': 'world'})
-    assert request.headers['content-type'] == 'application/json'
-
-def test_build_stream_request():
```

Comment:
nit: since `send_request` is still available on the client, you can keep at least some of these tests, to make sure that creating a request and sending it to `send_request` still works